README.md

```diff
@@ -44,6 +44,9 @@ This is the current hacker news ranking formula:
 The problem is that it only considers 1) **upvotes** and 2) **age**. It doesn't consider 3) **timing** or 4) **rank**. So a story that receives 100 upvotes at rank 1 is treated the same as one that receives 100 upvotes at rank 30. And upvotes received during peak hours US time are treated the same as upvotes received in the middle of the night.
 
 Our solution is to account for the effects of rank and timing, giving upvotes received at high ranks and peak times less weight.
+By doing this, the effects of the positive feedback loop are eliminated, and the best stories are given the attention they deserve.
+
+Note, that in this work we're not addressing the problem of overlooked stories. We'll approach this problem in the future.
```

Comment:
I think we can leave the comment about "overlooked stories" in the "Further Improvements" section, along with the issue about the causal model.